core/functions.php

```diff
@@ -452,9 +452,19 @@ function getActorThumbnail($name, $actorid = 0, $idSearchAllowed = true)
 	// identify actor by unique actor id, of by name
     if ($actorid && $idSearchAllowed) {
         $result = runSQL($SQL." WHERE actorid='".escapeSQL($actorid)."'");
-	}
-	if (!$actorid || (count($result) == 0)) {
```

Comment:
I would keep this bit and just add an is_array($result) check. No need for the is_countable bit...
is_countable is good for modern code but the runSQL function will only return an array (when all succeeds), so just check on that:
`if (!$actorid || (is_array($result) && count($result) == 0)) {`

Reply:
agree change in progress

Reply:
@johanneskonst I have made one small change to your suggestion, have group the is_array and count() in brackets for clarity.
if you agree I will do PR with squash and merge to clean up number of commits to master